internal/pkg/cli/clean/static_site.go

```diff
@@ -18,16 +18,16 @@ type bucketEmptier interface {
 	EmptyBucket(string) error
 }
 
-// StaticSite is used to clean up resources created for a static site.
-type StaticSite struct {
+// StaticSiteCleaner is used to clean up resources created for a static site.
+type StaticSiteCleaner struct {
 	app, env, svc        string
 	bucketResourceGetter bucketResourceGetter
 	bucketEmptier        bucketEmptier
 }
 
-// NewStaticSite returns an initialized static site.
-func NewStaticSite(app, env, svc string, rg bucketResourceGetter, emptier bucketEmptier) *StaticSite {
-	return &StaticSite{
+// StaticSite returns an initialized static site cleaner.
+func StaticSite(app, env, svc string, rg bucketResourceGetter, emptier bucketEmptier) *StaticSiteCleaner {
```

Comment:
Would `NewStaticSiteCleaner` make more sense for the function name🤔

Reply:
:sob: https://github.com/aws/copilot-cli/pull/4823#discussion_r1184413903

Reply:
:homer-disappearing: